demo/src/config/main-nav.js

```diff
@@ -0,0 +1,93 @@
+import LinkCreationModal from '../components/link-creation-modal';
+import LinkCreationModal2 from '../components/link-creation-modal-2';
+
+export default [
+  {
+    navType: 'item',
+    navId: 'Dashboard',
+    to: '/dashboard',
+  },
+  {
+    navType: 'dropdown',
+    navId: 'Samples',
+    dpLinks: [
+      {
+        to: '/alert-bars',
+        linkTitle: 'Alert Bars',
+      },
+      {
+        to: '/alerts',
+        linkTitle: 'Alerts',
+      },
+      {
+        to: '/badges',
+        linkTitle: 'Badges',
+      },
+    ],
+  },
+  {
+    navType: 'dropdown',
+    navId: 'Canvas',
+    dpLinks: [
+      {
+        to: '#',
+        linkTitle: 'Pages',
+      },
+      {
+        modal: LinkCreationModal,
```

Comment:
shouldnt this be a function? that way we can import the same function wherever we need it and put it as an onClick on a button